sematic/tests/data/test_settings.yaml

```diff
@@ -0,0 +1,8 @@
+active_profile: default
+profiles:
+  default:
+    SNOWFLAKE_USER: foobar
+  dev1: {}
+  dev2:
+    AWS_S3_BUCKET: sematic-dev
+    KUBERNETES_NAMESPACE: dev2
```

Comment:
This requires a migration of all users profile, and is not backward compatible. Can we let the provides at the root, and have `active_profile` at the root as well?

Reply:
Having the individual profiles at the root, together with the active profile would mean having a mixed type for the entries in the yaml, and custom handling for validation and interpretation.

I.e. instead of having
```python
@dataclass
class UserSettings:
    active_profile: str
    profiles: Dict[str, ProfileSettingsType]
```
we would need to have
```python
UserSettings = Dict[str, Union[ProfileSettingsType, str]]
```
and validation that exactly one of the values is a string.

Reply:
I'm with @tscurtu; migrating the existing `settings.yaml` files automatically seems like a fine solution to me and makes for a cleaner structure.